docs/components/DocumentationHeader.js

```diff
@@ -54,39 +53,39 @@ const STYLES_NAV = css`
   align-items: center;
   justify-content: space-between;
   position: relative;
-  background-color: white;
+  background-color: ${Constants.expoColors.white};
   z-index: 2;
   margin: 0 auto;
   padding: 0 16px;
   height: 60px;
-  width: auto;
-  max-width: 1440px;
   box-sizing: unset;
-
-  @media screen and (max-width: ${Constants.breakpoints.mobileStrict}) {
+  @media screen and (max-width: ${Constants.breakpoints.mobile}) {
     border-bottom: 1px solid ${Constants.expoColors.gray[250]};
   }
 `;
 
 const STYLES_MOBILE_NAV = css`
   padding: 0px;
   height: 56px;
+  background: ${Constants.expoColors.white};
+  display: none;
 
-  @media screen and (min-width: ${Constants.breakpoints.mobileStrict}) {
-    display: none;
+  @media screen and (max-width: ${Constants.breakpoints.mobile}) {
+    display: block;
```

Comment:
```suggestion
    display: flex;
```
The searchbar on the mobile nav is not vertically centered. If we want to keep `display: block`, I can look at some other ways to refactor.